compose/src/main/kotlin/com/yourssu/handy/compose/Chip.kt

```diff
@@ -0,0 +1,111 @@
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.wrapContentWidth
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import com.yourssu.handy.compose.foundation.*
+import com.yourssu.handy.compose.icons.HandyIcons
+import com.yourssu.handy.compose.icons.line.*
+
+enum class ChipSize(
+    internal val chipHeight: Dp
+) {
+    Large(32.dp),
+    Small(24.dp)
+}
+
+enum class ChipState {
+    Unselected,
+    Selected,
+    Disabled
+}
+
+@Composable
+fun Chip(
+    text: String = "",
+    modifier: Modifier = Modifier,
+    chipState: ChipState = ChipState.Unselected,
+    chipSize: ChipSize = ChipSize.Large,
+    leadingIcon: ImageVector? = HandyIcons.Line.InfoCircle,
+    trailingIcon: ImageVector? = HandyIcons.Line.Close,
+    chipOnClick: (() -> Unit)? = null,
+    leadingIconOnClick: (() -> Unit)? = null,
+    trailingIconOnClick: (() -> Unit)? = null
+) {
+
+    val backgroundColor = remember(chipState) {
+        when (chipState) {
+            ChipState.Unselected -> ColorGray100
+            ChipState.Selected -> ColorViolet100
+            ChipState.Disabled -> ColorGray50
+        }
+    }
+
+    val textColor = remember(chipState) {
+        when (chipState) {
+            ChipState.Unselected -> ColorGray700
+            ChipState.Selected -> ColorViolet500
+            ChipState.Disabled -> ColorGray300
+        }
+    }
+
+    val textStyle: HandyTextStyle =
+        if (chipState == ChipState.Selected) HandyTypography.B3Sb14 else HandyTypography.B3Rg14
+
+    Box(
+        modifier = modifier
+            .wrapContentWidth(unbounded = true)
+            .height(chipSize.chipHeight)
+            .background(backgroundColor, shape = RoundedCornerShape(16.dp))
+            .clickable(
+                enabled = chipState != ChipState.Disabled,
+                onClick = { chipOnClick?.invoke() })
+            .padding(horizontal = 12.dp),
+        contentAlignment = Alignment.Center
+    ) {
+        Row(
+            verticalAlignment = Alignment.CenterVertically,
+            horizontalArrangement = Arrangement.spacedBy(4.dp)
+        ) {
+            if (leadingIcon != null)
+                Icon(
+                    leadingIcon,
+                    tint = textColor,
+                    iconSize = IconSize.XS,
+                    modifier = Modifier.clickable(
+                        enabled = chipState != ChipState.Disabled,
+                        onClick = { leadingIconOnClick?.invoke() })
```

Comment:
```suggestion
                    modifier = Modifier.clickable(
                        enabled = chipState != ChipState.Disabled,
                        onClick = { leadingIconOnClick() },
                        interactionSource = remember { MutableInteractionSource() },
                        indication = null
                    )
```

interactionSource 랑 indication 을 위와같이 지정하면 클릭 시 눌리는 느낌의(ripple효과라고 합니다)동작이 없어집니다!
이런 효과들은 따로 문서에 기재되어있지 않은 이상 없게끔 해주시면 돼요!